pkg/network/usm/ebpf_main.go

```diff
@@ -267,10 +268,18 @@ func (e *ebpfProgram) initPrebuilt() error {
 	}
 	defer bc.Close()
 
-	var offsets []manager.ConstantEditor
-	if offsets, err = offsetguess.TracerOffsets.Offsets(e.cfg); err != nil {
+	var constants []manager.ConstantEditor
+	offsets, err := offsetguess.TracerOffsets.Offsets(e.cfg)
+	if err != nil {
 		return err
 	}
+	constants = append(constants, offsets...)
+
+	begin, end := network.EphemeralRange()
+	offsets = append(offsets,
+		manager.ConstantEditor{Name: "ephemeral_range_begin", Value: uint64(begin)},
+		manager.ConstantEditor{Name: "ephemeral_range_end", Value: uint64(end)},
+	)
```

Comment:
isn't that redundant? you're adding it to `init` as well

Reply:
yup, my bad. I removed it